grpc-protocol/src/main/java/com/linecorp/armeria/common/grpc/protocol/ArmeriaMessageFramer.java

```diff
@@ -145,20 +143,19 @@ public HttpData writePayload(ByteBuf message, boolean webTrailers) {
 
             final ByteBuf maybeEncodedBuf;
             if (encodeBase64) {
-                final ByteBuffer base64Encoded = Base64.getEncoder().encode(buf.nioBuffer());
+                final ByteBuf base64Encoded = Base64.encode(buf);
                 buf.release();
```

Comment:
Could we release the buffers at the `finally` block, so that any exceptions do not leak the buffers?